<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QMainWindow" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1923</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laser-Camera Calibration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QGLViewer" name="GLviewer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="ViewportSeparator">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ImageViewer" name="imageViewer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ToolbarSection">
      <item>
       <layout class="QHBoxLayout" name="Toolbar3D">
        <item>
         <widget class="QPushButton" name="PointPick">
          <property name="text">
           <string>Pick Points</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="ToolbarDivider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Toolbar2D">
        <item>
         <widget class="QPushButton" name="RectifyButton">
          <property name="text">
           <string>Recify Image</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PixelPick">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Pick Pixels</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushOptimize">
      <property name="text">
       <string>Calibrate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="consoleLog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1923</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionAbrir_Imagem"/>
    <addaction name="actionSet_camera_model"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Open point cloud...</string>
   </property>
  </action>
  <action name="actionAbrir_Imagem">
   <property name="text">
    <string>Open image...</string>
   </property>
  </action>
  <action name="actionSet_camera_model">
   <property name="text">
    <string>Set camera model...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGLViewer</class>
   <extends>QWidget</extends>
   <header location="global">qgl_viewer.h</header>
  </customwidget>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header>image_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
